project.toml

```diff
@@ -0,0 +1,40 @@
+[build-system]
```

Comment:
hey @AlexanderJuestel this file should be called pyproject.toml rather than project.toml :)

Reply:
I knew it looked different to other files